src/NUnitFramework/framework/Attributes/RangeAttribute.cs

```diff
@@ -23,15 +23,26 @@
 
 using System;
 using System.Collections;
+using System.Reflection;
+
+using NUnit.Compatibility;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
 
 namespace NUnit.Framework
 {
     /// <summary>
     /// RangeAttribute is used to supply a range of values to an
     /// individual parameter of a parameterized test.
     /// </summary>
-    public class RangeAttribute : ValuesAttribute
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true, Inherited = false)]
```

Comment:
Sorry - I missed your comment earlier. I think you're right, I'm not sure what `Inherited` does on a Parameter attribute either - I hadn't considered that this attribute was parameter only! I wouldn't worry about reverting it though, at least it's consistent now 🙂

Reply:
👍 FWIW, I pulled my branch down on my work computer where I have Resharper. It told me that setting the `Inherited` property was redundant here, so it sounds like it has no effect on parameter only attributes. Like you said, it's consistent now, so I left it in. It should be pretty trivial to clean all of them up in the future if that is desired.

Reply:
Ah cool. Sorry about that!